MDO/client-assets/assets/configurations/feature/apps.json

```diff
@@ -371,7 +370,7 @@
       "status": "live",
       "toolTip": "MDO",
       "shortName": "MDP",
-      "url": "https://mdo.igot-dev.in/"
+      "url": "https://sphere.aastrika.org/"
```

Comment:
This URL for mdo is wrong.. Please take a pull again and then merge with ur current changes,

Reply:
Do review now